src/Lykke.Service.HFT/Startup.cs

```diff
@@ -56,7 +56,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
                 services.AddSwaggerGen(options =>
                 {
-                    options.DefaultLykkeConfiguration("v1", "HighFrequencyTrading API");
+                    options.DefaultLykkeConfiguration("v1", "High-frequency trading API");
```

Comment:
Core.Constants.ComponentName instead?

Reply:
Wouldn't it be better to make "v1" a constant and use it here and below?

Reply:
"v1" as constant — fixed;
Core.Constants.ComnonentName is not really appropriate here, since it is a service name "Lykke.Service.HighFrequencyTrading", but here I wanted to use more friendly name.